docs/modules/ROOT/pages/Installation_IDE_Support.adoc

```diff
@@ -1,5 +1,4 @@
 = Installation and IDE Support
-:page-aliases: Installation.adoc, IDE_Support.adoc
```

Comment:
Those aliases are meant to keep older deep links working. I think we should keep them as they don't harm.